src/ResourceManager/DataFactoryV2/Commands.DataFactoryV2.Test/ScenarioTests/IntegrationRuntimeTests.ps1

```diff
@@ -124,11 +140,10 @@ function Test-SsisAzure-IntegrationRuntime
             -NodeCount 1 `
             -CatalogServerEndpoint $catalogServerEndpoint `
             -CatalogAdminCredential $mycreds `
-            -CatalogPricingTier 'S1' `
+            -CatalogPricingTier 'Basic' `
             -MaxParallelExecutionsPerNode 1 `
             -LicenseType LicenseIncluded `
             -Edition Enterprise `
-            -SetupScriptContainerSasUri $sasuri `
```

Comment:
Why are you no longer testing this?

Reply:
@zhangyd2015

Reply:
This is a quite low frequency usage scenario and quite complicated setup to run, no need to cover it here.